### DISCUSSION TITLE
按项目填好porkbun后报接口错误

### DISCUSSION THREAD

**Discussion by Ren9527** [jeessy2/ddns-go#523] [General]
请求接口 https://porkbun.com/api/json/v3/dns/retrieveByNameType/webren.pub/A/www 失败! 返回内容: {"status":"ERROR","message":"Invalid API key. (002)"} ,返回状态码: 400

**Comment by jeessy2** [comment]
都提示你：Invalid API key

**Reply by Ren9527** [reply]
额，大佬抱歉啊。我菜鸡一个，我把两个key都按对应填上了啊，还是报错。。。
已经用回明文Callback模式了。

**Reply by jeessy2** [reply]
重新创建key，还要在域名中启用API ACCESS
